runcoms/README.md

```diff
@@ -10,61 +10,199 @@ in */etc* by default.
+|                  | `/etc/zshrc`                            |                                   | system provided startup script, for interactive shells, (local shells that live in an `XTerm`, `URxvt`, `gnome-terminal`) or subshells (like running `xterm` from the command prompt of a _login xterm_, | site login accounting, security monitors, site command logging policy
+|                  |                                         | `${ZDOTDIR:-$HOME}/.zshrc`        | user customization of their *interactive* environment for terminals                                                                                                                                      | sourcing custom completions (`gcloud.comp.inc`), custom user frameworks (zprezto is invoked here), additional path required by custom software installed in `/opt/*`
+|                  | `/etc/zshlogin`                         |                                   | additional customization for interactive **login** shells, such as those accessed via ssh                                                                                                                | tmux attach to existing session
+|                  |                                         | `${ZDOTDIR:-$HOME}/.zshlogin`     | user script for *login shell startup* (this shell is a session leader, and allocates a `ptty`                                                                                                            | aliases, keybindings, personal startup programs, time tracking apps, session managers, quote of the day, fortunes
+| ..               | ... your shell session happens here ... |
+|                  |                                         | `${ZDOTDIR:-$HOME}/.zshlogout`    | personal cleanup tasks                                                                                                                                                                                   | setting personal `crontabs` or `at` tasks, logging checout to your time tracking software, fortune to say goodbye.
+| `/etc/zshlogout` |                                         |                                   | site cleanup tasks                                                                                                                                                                                       | site command logging policy stop, login accounting records, security context spindown, etcThe configuration files are read in the following order:
+
+Discussion
```

Comment:
Quite a few of the comments in this section are very useful, but I feel like most people are going to ignore a large block of text this big. Is there any way to pare this down a bit to focus on the recommendations and why they're being recommended? Perhaps even moving most of the recommendations to the table and mainly giving explanations here?

EDIT: After looking at the conclusion later, it seems like this was written as an essay, examining the different zsh configuration files and determining why prezto did things the way it did... and it does a good job of that. However, I'd still prefer to see slightly more focused documentation.

Reply:
Prezto's repo has no wiki enabled. if you wish you can add a link to the original write-up (with corrections coming, i'm still working a bit though) so that way any reference material that may be of help to users is still tied to the repo. I'll be adding that on my fork's wiki in case you want to add a wik to the official repo, we can just move it over here =) Here's the link to the wiki article:

https://github.com/hlecuanda/prezto/wiki/Prezto-Startup-Discussion

Reply:
on an unrelated note.. I remember you mentioning that you wanted to start clearing the backlog on PRs and Issues. I'll try and _triage_ the oldest PRs that are likely to contain conflicts from recently updated changes and try to resolve them so merging back to master won't be too much of a hassle. Unless you have a better suggestion of where to start? I'll be happy to go through that. may not be as glamorous as coming up with new code, but someone has to do it right? haha Let me know :smiley: